app.py

```diff
@@ -0,0 +1,13 @@
+import requests
+
+
+def get_name(n: int) -> str:
+    """ Get a random name """
+    for _ in range(n):
+        response = requests.get('https://api.namefake.com/')
+        yield response.json()['name']
```

Comment:
Can we add error handling to ensure that the response was made successfully?
```suggestion
        response = requests.get('https://api.namefake.com/')
        if response.ok:
            yield response.json()['name']
        else:
            response.raise_for_status()
```